### ISSUE TITLE
Supporting server rendering

### ISSUE THREAD

**Issue by phanviet** [mrsweaters/mithril-rails#13]
I have found https://github.com/StephanHoyer/mithril-node-render supports mithril server rendering. Could you integrate it (like react-rails worked) to rails server.

**Comment by mrsweaters** [comment]
The views are rendered server side. That's what this line is doing: https://github.com/mrsweaters/mithril-rails/blob/master/lib/mithril_rails/msx.rb#L19

**Comment by phanviet** [comment]
Yes, It's a msx assets pipeline engine. But for SEO, I want the views are displayed on the first load. In react-rails, it uses "prerender" parameter:
`<%= react_component('HelloMessage', {name: 'John'}, {prerender: true}) %>`
Would you integrate it in the future?
